assets/css/styles.css

```diff
@@ -0,0 +1,95 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    /* border: solid black 1px; */
+}
+
+.title{
+    text-align: center;
+    border: none;
+    padding: 50px 10px;
+}
+
+.main-container{
+    display: flex;
+    flex-direction: row;
+}
+
+.aside{
+    padding-left: 30px;
+    border-right: black solid 3px;
+}
+
+#city-input{
+    margin: 5px;
+}
+
+.submit{
+    margin-left: 15px;
+}
+
+ul{
+    list-style: none;
+}
+
+#search-btn{
```

Comment:
It's best to use class names for CSS if possible, rather than ID

Reply:
Will do :)